scene/resources/dynamic_font.cpp

```diff
@@ -795,18 +792,22 @@ int DynamicFont::get_spacing(int p_type) const {
 }
 
 void DynamicFont::set_spacing(int p_type, int p_value) {
+#define SET_SPACING_VALUE(NAME) \
+	spacing_##NAME = p_value;   \
+	emit_changed();             \
+	_change_notify("extra_spacing_" #NAME);
+
```

Comment:
In this trivial case, the define is probably not needed. It only saves a few lines of code and the code would be more easily readable without it.

Reply:
I was a bit on the fence about this myself. The main reason for this was that seemingly `_change_notify` was always called before `emit_changed`, but looking at other resources it's not a requirement.

I'll push the change after testing it to make sure. It seems current master broke GDScript's `tool` which I'm relying on for testing.

Reply:
Even it `emit_changed()` should be kept first, it's ok to have it duplicated in the four branches.

Reply:
Changed it and rebased the branch to an older commit from `master` as #40598 doesn't yet have support for `tool` which is rather useful for testing here.